website/content/en/docs/building-operators/ansible/installation.md

```diff
@@ -9,14 +9,15 @@ weight: 1
 Follow the steps in the [installation guide][install-guide] to learn how to install the `operator-sdk` CLI tool.
 
 ## Additional Prerequisites
-
+- [python][python] version 3.8.6+
```

Comment:
Are there breaking changes and/or features between 3.6 and 3.8 that would require users to use 3.8+?

Reply:
The main feature here is that 3.8+ supports a local dev environment and lots of security and performance updates. In this way, it eases local development and allows you to use Python modules that use 3.7+ language features described in https://docs.python.org/3/whatsnew/3.7.html and https://docs.python.org/3/whatsnew/3.8.html.

To be clear: we rely heavily on Ansible custom modules (i.e. Python modules) to fine-tune our Ansible operators. As such, we require it for our development process.

Reply:
Building the Ansible operator locally with Python versions older than 3.8 will most probably work, but the point is reproducibility: a local development environment that mirrors the one you build in the Dockerfile

Reply:
Does this answer your question?

Reply:
Yes thanks!